### ISSUE TITLE
Hidden Message checkbox in firefox browser

### ISSUE THREAD

**Issue by johnbritto1** [NarrativeScience/log.io#246]
Hello,

I tested the webserver (localhost:6688) on Firefox Browser 78.0.1 (64bit) in Linux Mint 19.3 Tricia but couldn't see the stream checkbox. It is appearing in google chrome browser.

![Capture](https://user-images.githubusercontent.com/5947168/88620937-dab5d480-d0bc-11ea-9fe0-4289b7fa4f5f.PNG)

**Comment by msmathers** [comment]
Click the "+ Add Screen" button and a screen will appear, along with a corresponding checkbox.

**Comment by msmathers** [comment]
If that doesn't work, could you please open up the browser console in Firefox and screenshot any errors you see.

**Comment by SodexisTeam** [comment]
Yes, With firefox, if we click on the "+Add Screen" button, then the checkbox appears. 
With Chrome, the screen appears by default. but not with the firefox. 

Thanks.

**Comment by johnbritto1** [comment]
Yes, With firefox, if we click the "+Add Screen" button, then the checkbox appears.
With Chrome, the screen appears by default. but not with the firefox.

Thanks.
